specification/cognitiveservices/data-plane/TranslatorText/preview/v1.0-preview.1/TranslatorBatch.json

```diff
@@ -958,10 +958,76 @@
         "Succeeded",
         "Failed",
         "Cancelled",
-        "Cancelling"
+        "Cancelling",
+        "ValidationFailed"
       ],
       "type": "string"
     },
+    "ErrorCodeV2": {
+      "description": "Enums containing high level error codes.",
+      "enum": [
```

Comment:
Is this enumeration fixed or is it likely to have new values added over time?  If it's the latter, please add the [`x-ms-enum`](https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-enum) annotation and set `modelAsString` to `true` to indicate this is an extensible enum.

Reply:
thanks.  we'll work on these changes